### PR TITLE
Release notes final touches

### PR DESCRIPTION
**Pull request by hythloda**
### Context
Finishing up the feedback cycle from google doc

### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [x] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [x] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [x] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation

### CODE REVIEW

**Comment by hythloda** [comment]
@bot-gradle test bd

**Review by tresat — APPROVED** [review]

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -168,7 +167,7 @@ See the [`kotlin-dsl` plugin manual](userguide/kotlin_dsl.adoc#sec:kotlin-dsl_pl
 
 Gradle 8.0 introduces an interpreter for the [declarative plugins {} blocks](https://github.com/gradle/gradle/blob/release/subprojects/docs/src/docs/release/userguide/plugins.html#sec:constrained_syntax) in `.gradle.kts` scripts that make the overall build time around 20% faster. By default, calling the Kotlin compiler for declarative `plugins {}` blocks is avoided.
 
-To utilize this performance, ensure you are using the supported formats in the declarative `plugins {}` blocks:
+To utilize this performance, ensure you are using the supported formats in the declarative `plugins {}` blocks, for example:
```
Comment (ljacomet) [inline subprojects/docs/src/docs/release/notes.md:170]: I find the start of the sentence weird.

What about:
> To benefit from this performance improvement, ensure ...

**Review by ljacomet — APPROVED** [review]
See for a suggestion below.

If you disagree, this is good to go!

**Comment by hythloda** [comment]
@bot-gradle test BD

**Comment by hythloda** [comment]
@bot-gradle test and merge

**Pull request merged**